packages/generator/src/v2.0/generateClientSdkVersionTwoPointZero.test.ts

```diff
@@ -642,7 +645,10 @@ describe("generator", () => {
         }
       }
 
-      /** @deprecated Use \`markTodoCompleted.Definition\` **/
+      /**
+       * An action which takes different types of parameters
+       * @param {ActionParam.ObjectType<Todo>} [object] A Todo to mark completed
+       */
```

Comment:
Don't you want this on the const not this object? When people hover over `client(markTodoCompleted)` they arent seeing this type...